packages/fx-core/src/core/middleware/MigrationUtils.ts

```diff
@@ -134,6 +134,8 @@ export const fixedNamingsV3: { [key: string]: string } = {
   "state.fx-resource-aad-app-for-teams.oauthAuthority": "AAD_APP_OAUTH_AUTHORITY",
   "state.fx-resource-bot.botId": "BOT_ID",
   "state.fx-resource-bot.botPassword": "SECRET_BOT_PASSWORD",
+  "state.fx-resource-frontend-hosting.sslCertFile": "SSL_CRT_FILE",
```

Comment:
@swatDong is these expected env output from local debug?

Reply:
I think yes.